functional-test/pages/death.of.appelant.ts

```diff
@@ -39,38 +43,7 @@ export class DeathOfAppellantPage {
             await webAction.verifyPageLabel('//h2[.=\'Address Details\']', deathOfAnAppellant.addressDetailsSectionHeading);
             await webAction.verifyPageLabel('//h2[.=\'Contact Details\']', deathOfAnAppellant.contactDetailsSectionHeading);
 
-            await webAction.verifyPageLabel('[for=\'appeal_appellant_appointee_name_title\'] > .form-label', deathOfAnAppellant.titleTextFieldLabel);
-            await webAction.verifyPageLabel('[for=\'appeal_appellant_appointee_name_firstName\'] > .form-label', deathOfAnAppellant.firstNameTextFieldLabel);
```

Comment:
Can I assume all this code has been moved inside the helper...?

Reply:
@pats-john  Yes, code is now moved into Pages & tabs folder as part of refactoring.